storage/persistent_storage/persistent-storage-vsphere.adoc

```diff
@@ -27,7 +27,7 @@ requested by users.
 Dynamically provisioning VMware vSphere volumes is the recommended method.
 
 .Prerequisites
-An {product-title} cluster installed on a VMware vSphere version that meets the requirements for the components that you use. See xref:../../installing/installing_vsphere/installing-vsphere.adoc[Installing a cluster on vSphere]
+An {product-title} cluster installed on a VMware vSphere version that meets the requirements for the components that you use. See xref:../../installing/installing_vsphere/installing-vsphere.adoc[Installing a cluster on vSphere].
```

Comment:
Prereqs should always be bulleted, even if there is only one. See https://redhat-documentation.github.io/modular-docs/#procedure-module-guidelines.

Also suggest making the "See xref" a fuller sentence, like "See xref for more information/instructions" or something.